docs/mockwbemserver.rst

```diff
@@ -851,50 +853,48 @@ that somewhere an instance the class must be created each time a namespace is
+specific provider classes and registering these subclasses as providers using
 :meth:`~pywbem_mock.FakedWBEMConnection.register_provider`.
 
-Pywbem defines user-defined provider types so that specific request operations
-can be executed for each provider type.
+The WBEM operations supported by user-defined providers can be implement one by
+one. If a user-defined provider does not implement all WBEM operations, the
+default implementation is used.
 
+The following table shows the WBEM operations for which user-defined providers
+are supported, and the corresponding provider types:
 
-.. table:: Pywbem_mock provider types
+.. table:: WBEM operations of user-defined providers
 
-    ====================== =================== ======================== ================================
-    Provider Type          type name           CIM Request Operations   Default provider class
-    ====================== =================== ======================== ================================
-    method                 "method"            InvokeMethod             :ref:`Method Provider`
-    instance write         "instance-write"    CreateInstance           :ref:`Instance Write Provider`
-    instance write         "instance-write"    ModifyInstance           :ref:`Instance Write Provider`
-    instance write         "instance-write"    DeleteInstance           :ref:`Instance Write Provider`
-    ====================== =================== ======================== ================================
-
-Each user-defined provider is a Python class which is a subclass
-of the corresponding default provider for the provider type as defined in
-the previous table.
+    ============== ================ ============== ==============================
+    Provider type  Type name        WBEM operation Details
+    ============== ================ ============== ==============================
+    method         "method"         InvokeMethod   :ref:`Method Provider`
+    instance write "instance-write" CreateInstance :ref:`Instance Write Provider`
+    instance write "instance-write" ModifyInstance :ref:`Instance Write Provider`
+    instance write "instance-write" DeleteInstance :ref:`Instance Write Provider`
+    ============== ================ ============== ==============================
 
 
 .. _`Creating user-defined instance providers`:
 
 Creating user-defined instance providers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
- User-defined instance providers can override the default implementations for
- the methods defined in the above table (``CreateInstance``, ``ModifyInstance``,
- and ``DeleteInstance``). These methods are defined in a user-defined class
- that subclasses :class:`pywbem_mock.InstanceWriteProvider`. The details of
- defining this provider are in section :ref:`Instance Write Provider`.
+
+User-defined instance write providers can override the default implementations for
+the methods defined in the above table (``CreateInstance``, ``ModifyInstance``,
```

Comment:
Shouldn't the work methods really be "operations"?

Reply:
Yes. DONE